chrome/reportableUrl.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import isReportableURL from "../shared/base.js";
```

Comment:
What's the purpose of having a new reportableUrl.js file in each of the addon source directories? We already have the method defined in the shared base.